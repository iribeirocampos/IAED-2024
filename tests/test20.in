p teste1 20 0.3 0.5 5
e teste1 11-JJ-11 01-01-2024 10:00
s teste1 11-JJ-11 01-01-2024 11:00
e teste1 11-JJ-11 01-01-2024 11:00
s teste1 11-JJ-11 01-01-2024 13:00
e teste1 11-JJ-11 01-01-2024 13:00
s teste1 11-JJ-11 02-01-2024 13:00
e teste1 11-JJ-11 02-01-2024 13:00
s teste1 11-JJ-11 06-01-2024 13:00
e teste1 11-JJ-11 06-01-2024 13:00
s teste1 11-JJ-11 07-01-2024 10:00
e teste1 11-JJ-11 07-01-2024 13:00
s teste1 11-JJ-11 08-01-2024 15:00
e teste1 11-JJ-11 08-01-2024 15:00
s teste1 11-JJ-11 18-01-2024 17:00
q
