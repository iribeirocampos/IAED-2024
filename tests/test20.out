teste1 19
11-JJ-11 01-01-2024 10:00 01-01-2024 11:00 1.20
teste1 19
11-JJ-11 01-01-2024 11:00 01-01-2024 13:00 3.20
teste1 19
11-JJ-11 01-01-2024 13:00 02-01-2024 13:00 5.00
teste1 19
11-JJ-11 02-01-2024 13:00 06-01-2024 13:00 20.00
teste1 19
11-JJ-11 06-01-2024 13:00 07-01-2024 10:00 5.00
teste1 19
11-JJ-11 07-01-2024 13:00 08-01-2024 15:00 8.20
teste1 19
11-JJ-11 08-01-2024 15:00 18-01-2024 17:00 53.20
